configscanning/comparefiles.py

```diff
@@ -0,0 +1,124 @@
+import argparse
+import difflib
+import glob
+import logging
+import os
+
+import boto3
+
+from configscanning import k8sutils
+
+parser = argparse.ArgumentParser()
```

Comment:
just a general comment, don't rewrite this, but the click library is great for creating CLIs and makes documenting them very easy. I would suggest to use it for future CLI apps.